CHANGELOG.md

```diff
@@ -73,4 +76,14 @@ parameter is provided in the initial request, `V2PaginationImpl` will continue t
 all pages until the number of elements in the response no longer equals the expected count. 
 This will avoid infinitely looping over pages until an empty page is discovered.
 
-## 1.0.6 (Work in progress)
+## 1.0.7 (September 17, 2020)
+* UGCShareConnection.retrieveUGCPostsByAuthors() now includes sortBy param to get posts
+sorted by creation date.
+
+## 1.0.8 (October 15, 2020)
+* OrganizationConnection.retrieveOrganizationBrand will return an organization brand
+ based on the organization URN.
+ OrganizationConnection.retrieveOrganizationBrandByParentOrganization will return 
```

Comment:
Should this have a new list item here? Not sure if it is important enough to delay the merge
```suggestion
* OrganizationConnection.retrieveOrganizationBrandByParentOrganization will return 
```